### DISCUSSION TITLE
generate/app-descriptor :: Parameter not valid: adMobAndroidApplicationId=

### DISCUSSION THREAD

**Discussion by mgravano** [airsdk/apm#158] [General]
Hi,
I am using apm, following all the steps.
When I want to generate the app-descriptor in my Terminal, it tells me:
generate/app-descriptor :: Parameter not valid: adMobAndroidApplicationId=
What am I doing wrong? Could you help me?
Thank you!

**Comment by marchbold** [comment]
This message indicates you haven't provide a configuration value for one of the extensions you are using. In particular that's for the admob android application id.

If you run the following:

```
apm project config set
```

it will run through all the configuration values you need to set. 

You can run the following to see the current values:

```
apm project config
```

**Comment by mgravano** [comment]
Thank you, Michael!
